boilerPlate/src/app/shared/components/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,51 @@
+<!-- component -->
+<div (click)="open = false"
+  class="flex flex-col  md:w-64 text-gray-700 bg-white dark-mode:text-gray-200 dark-mode:bg-gray-800 flex-shrink-0">
+  <div class="flex-shrink-0 px-8 py-4 flex flex-row items-center justify-between">
+    <a href="javacript:void(0)"
+      class="text-lg font-semibold tracking-widest text-gray-900 uppercase rounded-lg dark-mode:text-white focus:outline-none focus:shadow-outline">Flowtrail
+      UI</a>
+    <button class="rounded-lg md:hidden rounded-lg focus:outline-none focus:shadow-outline" (click)="open = !open">
```

Comment:
We can have shared Button components with a specific theme that can be used in any project by just changes themes,
and it can have types like primary and secondary.

`<Button type="primary">text</Button>`
`<Button type="secondary">text</Button>`

Reply:
I think this won't be beneficial for the user to customize more they will be restricted to use our define button types